module/VuFind/src/VuFind/DigitalContent/OverdriveConnector.php

```diff
@@ -449,6 +449,10 @@ public function doOverdriveCheckout($overDriveId)
             return $result;
         }
         if ($config = $this->getConfig()) {
+            if (!$config->usePatronAPI) {
+                $this->error('Checkout - Overdrive patron APIs are disabled.');
+                return $result;
+            }
```

Comment:
This code will presumably never be reached since the UI would now not present options to do a checkout; same with most of the other mutation functions below.  But I think it's better to catch the scenario just in case.